grade-scripts/trackers-from-requests.js

```diff
@@ -51,23 +49,44 @@ const run = async () => {
         let trackersBlocked = {}
         let trackersNotBlocked = {}
         let totalBlocked = 0
+        let requestsBlocked = []
+        let rulesUsed = {}
```

Comment:
All the changes in this file are just to collect data on what we're blocking or not blocking for use later on.